src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnector.java

```diff
@@ -45,9 +46,10 @@ public class SnowflakeSinkConnector extends SinkConnector {
   // create logger without correlationId for now
   private static LoggerHandler LOGGER = new LoggerHandler(SnowflakeSinkConnector.class.getName());
 
+  private final int closeConnectorRetryCount = 5;
```

Comment:
Why 5?

Reply:
arbitrary number i decided, happy to change it to anything else! closing it is best effort since an orphaned client shooould be garbage collected

Reply:
That's fine, was just curious.

Reply:
removed the retry because we retry in the client